src/Domain/CoordinateMaps/CoordinateMap.hpp

```diff
@@ -74,18 +70,22 @@ class CoordinateMapBase : public PUP::able {
   virtual tnsr::I<double, Dim, TargetFrame> operator()(
       tnsr::I<double, Dim, SourceFrame> source_point,
       double time = std::numeric_limits<double>::signaling_NaN(),
-      const std::unordered_map<std::string,
-                               domain::FunctionsOfTime::FunctionOfTime&>&
+      const std::unordered_map<
+          std::string,
+          std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>&
```

Comment:
Here it looks like you're replacing a map of non-owning references with a map of owning pointers. Can you comment (just here in Github, to help me understand) where the change in the call site is? I see the corresponding change in the test, but nowhere else. Is this functionality simply not used in the source code yet?

Reply:
Yes, so currently we don't have any moving  mesh evolutions, only some tests for the maps. What this PR is addressing is that when we add the functions of time into the DataBox we will add them as `unique_ptr`s through the base class, which means a more natural interface for these functions is to take the `unique_ptr`s rather than references. I'm currently working on getting the time dependence into the evolution executables.